website/static/js/filesWidget.js

```diff
@@ -0,0 +1,77 @@
+var $ = require('jquery');
```

Comment:
@lyndsysimon `fileWidget.js` and `registrationEditorExtensions.js` are used to embed the OSF file picker in the pre-reg questions